ext/blake2_ext/blake2-impl.h

```diff
@@ -1,20 +1,23 @@
 /*
```

Comment:
Can we build a rake task that we could run that would automate the vendoring of those files?

Also, we could just ignore them in the repo and download them when we release the gem... not sure what's the best. Thoughts?